contents/customers/speakeasy.md

```diff
@@ -0,0 +1,65 @@
+---
+title: How Speakeasy manages features and developer relations with PostHog
+customer: Speakeasy
+logo: ../images/customers/speakeasy/speakeasy-logo.png
+logoDark: ../images/customers/speakeasy/speakeasy-logo-original.svg
+featuredImage: ../images/customers/speakeasy/speakeasy-featured.png
+industries:
+    - Devtool
+users:
+    - Product
+    - Engineering
+    - Growth
+    - Developer Relations
+toolsUsed:
+    - Feature flags
+    - Product analytics
+    - Templates
+date: 2023-08-31
+---
+
+A team of engineers founded Speakeasy with a mission to enhance how engineers create and maintain their APIs. Dozens of teams, including Airbyte and Motion, now trust [Speakeasy](https://speakeasyapi.dev/) to provide everything engineers need to work with APIs: from native-language SDKs and AI-powered maintenance tools to world-class reference docs.
+
+Internally, the team relies on PostHog for both product development and marketing activities, a practice they've maintained since the early days.
+
+"At the beginning, we were trying to define our entire product operating system," says Nolan Sullivan, Speakeasy's founding developer relations expert. "We chose PostHog because it's open-source, and we really liked its appearance. Initially, we used the (now [sunset](/blog/sunsetting-helm-support-posthog)) self-hosted version, but later switched to the cloud version."
```

Comment:
product operating system!

Reply:
Ha, yes - and this is actually what they said too. It's made me reasses my initial judgement of the term, tbh. @corywatilo was right all along!